npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater.rb

```diff
@@ -605,12 +603,10 @@ def git_dependencies_to_lock
         def lock_deps_with_latest_reqs(content)
           json = JSON.parse(content)
 
-          NpmAndYarn::FileParser::DEPENDENCY_TYPES.each do |type|
-            json.fetch(type, {}).each do |nm, requirement|
-              next unless requirement == "latest"
+          NpmAndYarn::FileParser.each_dependency(json) do |nm, requirement, _type|
+            next unless requirement == "latest"
 
-              json[type][nm] = "*"
-            end
+            json[type][nm] = "*"
```

Comment:
```suggestion
            json[_type][nm] = "*"
```

Reply:
Sorbet win?

Reply:
Nice!

Will instead rename the `_block` variable, since the `_` prefix is usually aimed at unused variables.

But yess, yet another Sorbet win! 💪